runtime/doc/diagnostic.txt

```diff
@@ -163,6 +163,16 @@ DiagnosticSignHint
   Used for "Hint" signs in sign column.
   See |vim.diagnostic.set_signs()|
 
+==============================================================================
+AUTOCOMMANDS					*diagnostic-autocommands*
```

Comment:
let's try to use "events" where possible, autocmds imply vimscript but really they're just events and we will eventually have Lua handlers for them.